examples/sparse/sign.py

```diff
@@ -0,0 +1,111 @@
+"""
+[SIGN: Scalable Inception Graph Neural Networks]
+(https://arxiv.org/abs/2004.11198)
+
+This example shows a simplified version of SIGN: a precomputed 2-hops diffusion
+operators on top of symmetrically normalized adjacency matrix A.
+"""
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.optim import Adam
+
+from dgl.data import CoraGraphDataset
+from dgl.mock_sparse import create_from_coo, diag, identity
+
+################################################################################
+# (HIGHLIGHT) Take the advantage of DGL sparse API to significantly simplify the
+# code for sign diffusion operator.
+################################################################################
+def sign_diffusion(A, X, r):
+    # Perform the r-hops diffusion operation.
+    X_sign = [X]
+    for _ in range(r):
+        X = A @ X
+        X_sign.append(X)
+    return X_sign
+
+
+class SIGN(nn.Module):
+    def __init__(self, in_size, out_size, r, hidden_size=64):
+        super().__init__()
+        self.theta = nn.ModuleList(
+            [nn.Linear(in_size, hidden_size) for _ in range(r + 1)]
+        )
+        self.omega = nn.Linear(hidden_size * (r + 1), out_size)
```

Comment:
I never see people use `theta` and `omega` in defining a model class. If you really want to do that, leave a comment for the equation.